Cargo.toml

```diff
@@ -7,12 +7,30 @@ edition = "2021"
 
 [dependencies]
 crc32fast = "1.3.2"
+crossbeam = "0.8.2"
+chrono = "0.4.31"
+crossbeam-channel = "0.5.8"
+parking_lot = "0.12.1"
+hashbrown = "0.14.2"
+lru = "0.12.0"
 
 # below is for the async runtime
 async-task = "4.4.0"
 futures-lite = "1.12.0"
 once_cell = "1.17.1"
 flume = "0.11.0"
+bytes = "1.5.0"
 
 [dev-dependencies]
 tempdir = "0.3"
+rand = "0.8.5"
+criterion = "0.5.1"
+jemallocator = "0.3.2"
```

Comment:
jemallocator 0.3.2 seems pretty old (4 years ago),  currently 0.5.4

Reply:
Thanks for pointing it out!